### PR TITLE
Add pdvega to conda-forge

### PR DESCRIPTION
**Pull request by adbeda**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by adbeda** [comment]
@msarahan 
The build for OSX failed on this and 6 most recent other packages with error:
```
     The following packages will be SUPERSEDED by a higher-priority channel:

      pyopenssl                                       pkgs/main --> conda-forge


InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/cryptography-2.7-py37h212c5bf_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488145510576)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/yaml-0.1.7-h1de35cc_1001.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488130959504)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/ncurses-6.1-h0a44026_1002.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167094112)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/xz-5.2.4-h1de35cc_1001.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488130960064)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/urllib3-1.25.3-py37_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488130960624)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/setuptools-41.0.1-py37_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167095616)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/sqlite-3.29.0-hb7d70f7_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/run_exports.json (errno=22, retcode=-30, archive_p=140488154820064)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/six-1.12.0-py37_1000.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167091056)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/pycparser-2.19-py37_1.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167095328)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/ruamel_yaml-0.15.71-py37h1de35cc_1000.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488145503808)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/pysocks-1.7.0-py37_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488130961888)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/pyopenssl-19.0.0-py37_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167095888)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/tk-8.6.9-h2573ce8_1002.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/run_exports.json (errno=22, retcode=-30, archive_p=140488145496288)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/cffi-1.12.3-py37hccf1714_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167096448)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/readline-8.0-hcfe32e1_0.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/run_exports.json (errno=22, retcode=-30, archive_p=140488145497440)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/idna-2.8-py37_1000.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488154826144)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/chardet-3.0.4-py37_1003.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167096448)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/pycosat-0.6.3-py37h1de35cc_1001.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488154827872)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/libcxxabi-8.0.0-4.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488154828432)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/asn1crypto-0.24.0-py37_1003.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488167097040)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/zlib-1.2.11-h01d97ff_1005.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/run_exports.json (errno=22, retcode=-30, archive_p=140488167097600)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/libcxx-8.0.0-4.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488130966688)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/libffi-3.2.1-h6de7cb9_1006.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/run_exports.json (errno=22, retcode=-30, archive_p=140488154833456)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/python-3.7.3-h93065d6_1.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/no_link (errno=22, retcode=-30, archive_p=140488145489376)')
InvalidArchiveError('Error with archive /usr/local/miniconda/pkgs/requests-2.22.0-py37_1.tar.bz2.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCould not open info/hash_input.json (errno=22, retcode=-30, archive_p=140488154833456)')

##[error]Bash exited with code '1'.
##[section]Finishing: Configure conda and conda-build
```
I noticed the answer you gave [here](https://github.com/conda/conda/issues/8870#issuecomment-509276523). is there a fix to this?

**Comment by adbeda** [comment]
@conda-forge/help-python this recipe is ready for review. 

Thanks!

File: recipes/pdvega/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e7b3c3ad36f42e2289a93bca39336dd3bea2d24bcaf7323c4d5b8322fc3a017
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/pdvega/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Comment by adbeda** [comment]
@marcelotrevisani I often use `conda skeleton` to create recipes and the script for installing shows up as I originally submitted it. 
Thanks! Your feedback makes the recipe cleaner.

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani I often use `conda skeleton` to create recipes and the script for installing shows up as I originally submitted it.
> Thanks! Your feedback makes the recipe cleaner.

Yes, sorry about it! I am going to propose this change on conda skeleton

**Comment by adbeda** [comment]
Awesome! thanks

**Pull request merged**